Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1952,10 +1952,12 @@ private enum ExceptionMode {
                     PythonThread bpThread = null;
                     bool wrongLine = false;
                     proc.BreakpointHit += (sender, args) => {
+                        var continueEvent = new AutoResetEvent(true);

```

Comment:
It seems it's already scoped as narrow as it can possibly be (in the same scope where it's waited on).

I'll add a `using`, but note that pretty much none of these tests here do that for the events that they use. We should probably just clean it all up at some point.